content/docs/user-guide/assets/asset-processor/configuration.md

```diff
@@ -5,21 +5,77 @@ description: Configure Asset Processor for Open 3D Engine (O3DE) with AssetProce
+
+### Scan Folders
 
-* Set the number of process jobs.
-* Specify *metafile fingerprinting*, that is, which files trigger related files to be rebuilt.
+Scan folders can be added as registry settings. In most cases you'll use the asset folders of Gems or your project, which will be automatically added as scan folders. However, if you need to add a new scan folder for tracking source assets, you can add it under the registry setting path `Amazon/AssetProcessor/Settings/` with a pattern of:
+
+```json
+"ScanFolder <scan folder name>": {
+  "watch": "<scan folder path>",
+  "display": "<Display name for the scan folder>",
+  "recursive": <0|1>, // 1 scans all subdirectories for assets recursively, 0 only includes assets in this folder and not subfolders.
+  "order": <any number> // The order this scan folder is handled compared to other scan folders.
+}
+```
+
+Scan folder order is used when two assets have the exact same relative path to the scan folder root. In these situations, the source asset in the scan folder with the higher priority will be used over the other one.
+
+### Excluding Folders
 
-The `Registry/AssetProcessorPlatformConfig.setreg` is well commented and has many syntax examples for the available configuration options. To add project-specific overrides, you can add an `AssetProcessorGamePlatformConfig.setreg` file to your projects’s `Registry` directory. This file is read after the engine `AssetProcessorGamePlatformConfig.setreg` configuration file and can have additional project-specific settings for the ignore list, platforms, and file types.
\ No newline at end of file
+The exclusion system is used to block files matching patterns from asset processing. These exist under the registry path `Amazon/AssetProcessor/Settings/Exclude` with a pattern that looks like:
+
+```json
+"Exclude <exclusion title>": {
+  "pattern": "<regex of files to exclude>"
+}
+```
+
+### Asset Processing Tags
+
+Each platform has a collection of tags applied to it, under the registry path `Amazon/AssetProcessor/Settings/`. These tags are checked by other systems for processing, but don't directly trigger any activity themselves. Platform tags follow the pattern:
+
+```json
+"Platform <platform name>": {
+  "tags:" "<comma-separated list of tags>"
+}
+```
+
+### Copy Jobs
+
+Copy jobs are for source assets that should be copied directly to the cache as a product asset. Some assets don't need additional processing, so setting up a copy job this way is quicker than authoring an entire C++ based builder to make content of that type available to your game as an asset. Copy jobs are managed by a registry setting pattern under `Amazon/AssetProcessor/Settings/`, using the following pattern:

```

Comment:
Maybe add line that you can also skip files on some platforms if they are not needed. See the skip param in the example below etc.

Personally I think its worth calling out how you skip processing as here as it can be pretty useful.